api/src/controllers/anime/updateAnime.ts

```diff
@@ -0,0 +1,25 @@
+import type { Request, Response, NextFunction } from '@@types/express.js';
+import { errors, entities } from '@@utils/index.js';
+import { Anime } from '@@entities/index.js';
+
+interface Params {
+    id: number
+};
+
+// Update
+async function update(req: Request, res: Response<Params>, next: NextFunction) {
+    const { params } = res.locals;
+    const { id } = req.params;
+
+    const [anime, animeErr] = await entities.update<Anime>(Anime, {
+        id: id
+    });
+
+    if(animeErr) {
```

Comment:
You can add a `errors.sendEntitiesResponse` here too so we don't potentially send an undefined value in the results